src/main/java/net/earthcomputer/clientcommands/command/CDebugCommand.java

```diff
@@ -1,34 +1,34 @@
 package net.earthcomputer.clientcommands.command;
 
-import com.mojang.brigadier.Command;
-import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.*;
```

Comment:
I am sorry I was being ambiguous. I meant to say that you should use wildcard imports when you are importing things from DSL's like Brigadier, not everywhere. Even more so I recommend setting the threshold in your IDE for converting to wildcard imports to something like 25, the debug of 4 in IntelliJ is way too low.

Reply:
Wildcards for static imports only.

Reply:
Ok, thanks for the clarification!